bokehjs/src/coffee/api/models.coffee

```diff
@@ -81,20 +81,7 @@ module.exports = {
   LinearColorMapper:                      require("../models/mappers/linear_color_mapper").Model
 
   ## api/typings/models/markers.d.ts
-  Marker:                                 require("../models/markers/marker").Model
-  Asterisk:                               require("../models/markers/asterisk").Model
-  Circle:                                 require("../models/glyphs/circle").Model
```

Comment:
Good catch thanks 

> On Aug 23, 2016, at 03:40, Mateusz Paprocki <notifications@github.com> wrote:
> 
> In bokehjs/src/coffee/api/models.coffee:
> 
> > @@ -81,20 +81,7 @@ module.exports = {
> >    LinearColorMapper:                      require("../models/mappers/linear_color_mapper").Model
> >  
> >    ## api/typings/models/markers.d.ts
> > -  Marker:                                 require("../models/markers/marker").Model
> > -  Asterisk:                               require("../models/markers/asterisk").Model
> > -  Circle:                                 require("../models/glyphs/circle").Model
> This entry should be restored and moved up into the proper group.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>